README.md

```diff
@@ -6,36 +6,111 @@
 [![codecov](https://codecov.io/gh/ZcashFoundation/zebra/branch/main/graph/badge.svg)](https://codecov.io/gh/ZcashFoundation/zebra)
 ![License](https://img.shields.io/badge/license-MIT%2FApache--2.0-blue.svg)
 
-### 🚧 UNDER CONSTRUCTION 🚧
+## About
 
 [Zebra](https://zebra.zfnd.org/) is the Zcash Foundation's independent,
 consensus-compatible implementation of the Zcash protocol, currently under
 development.  Please [join us on Discord](https://discord.gg/na6QZNd) if you'd
 like to find out more or get involved!
 
+## Alpha Releases
+
+Our first alpha release is planned for December 8th, 2020.
+
+The goals of this release are to:
+- participate in the Zcash network,
+- replicate the Zcash chain state, and
+- implement the Zcash proof of work consensus rules.
+
+### Getting Started
+
+Run `cargo ...` **TODO**
+
+If you're interested in testing out zebrad please feel free, but keep in mind
+that there is a lot of key functionality still missing.
+
+### System Requirements
+
+**TBD**
```

Comment:
What does our CI currently run?

Reply:
cc @dconnolly, the long running containers are probably most relevant to our system requirements here